_shared/project/requirements/tests.in

```diff
@@ -1,8 +1,14 @@
 pip-tools
 pip-sync-faster
 -r prod.txt
-coverage[toml]
 pytest
+pytest-coverage
```

Comment:
We're now using `pytest-coverage` instead of using `coverage` directly. This is because `pytest-coverage` automatically works when running the tests in parallel with `pytest-xdist`, combining the coverage from each parallel test process.